libs/perun/openapi/src/lib/api.module.ts

```diff
@@ -49,7 +49,7 @@ import { VosManagerService } from './api/vosManager.service';
     VosManagerService ]
 })
 export class ApiModule {
-    public static forRoot(configurationFactory: () => Configuration): ModuleWithProviders<any> {
+    public static forRoot(configurationFactory: () => Configuration): ModuleWithProviders {
         return {
```

Comment:
Also this change is causing error: ` ERROR in ../libs/perun/openapi/src/lib/api.module.ts:52:71 - error TS2314: Generic type 'ModuleWithProviders<T>' requires 1 type argument(s).`

Reply:
This is weird, I don't know why this line has changed. Anyway, I have removed this change from the PR.